### PR TITLE
Fixes for PR 227

### PR DESCRIPTION
**Pull request by drpatelh**

### CODE REVIEW

File: nf-core-contributors.yaml [hunk 1]
```diff
@@ -435,3 +419,18 @@ contributors:
       contact_github: abhi18av
       location: [12.844722, 77.663056]
 
+    - full_name: Max Planck Institute for Biochemistry
+      short_name: MPIB
+      description: >
+        The Max Planck Institute of Biochemistry aims to understand the structure and function of proteins -
+        from single molecules to complex organisms.
+      affiliation: Technical University of Munich
+      address: Am Klopfersptiz 18, 82152 Planegg, Germany
```
Comment (apeltzer) [inline nf-core-contributors.yaml]:
```suggestion
      address: Am Klopferspitz 18, 82152 Planegg, Germany
```

Comment (apeltzer) [reply nf-core-contributors.yaml]: Typo Alert!

Comment (drpatelh) [reply nf-core-contributors.yaml]: Fixed 👍

Comment (drewjbeh) [reply nf-core-contributors.yaml]: Thanks!

**Pull request merged**